config/routes/data.rb

```diff
@@ -239,8 +239,11 @@
 
 resources :extracts do
   concerns [:data_routes]
-  resources :origin_relationships, shallow: true, only: [:index], defaults: {format: :json}
+  collection do
+    get :select_options, defaults: {format: :json}
+  end
 end
+resources :origin_relationships, shallow: true, only: [:index], defaults: {format: :json}
```

Comment:
Commenting this out resolves one spec failure. You meant to have this inside `resources :extracts` block @mjy? Looks it was actually taken out from it. If that was intentional then I guess best to do is remove the line altogether.

```
rspec ./spec/routing/origin_relationships_routing_spec.rb:6 # OriginRelationshipsController routing routes to #index
```